distribution/ddf-common/src/main/resources/common-bin.xml

```diff
@@ -58,6 +58,7 @@
 
                 <!-- Skipping customized libraries. -->
                 <exclude>**/org/apache/aries/blueprint/org.apache.aries.blueprint.core/1.8.2/org.apache.aries.blueprint.core-1.8.2.jar</exclude>
+                <exclude>**/org/eclipse/platform/org.eclipse.osgi/3.11.3/org.eclipse.osgi-3.11.3.jar</exclude>
```

Comment:
❓ Where is the forked code at?

Reply:
https://github.com/stustison/rt.equinox.framework/commit/76f64cc71ad002d1da0c7bd6cec81afa300d6346